README.md

```diff
@@ -13,6 +13,7 @@ YEAR=3  # desired year of the schedule (3, 4 or 5)
 DISCORD_TOKEN=XXX  # the token of the bot
 DISCORD_CHANNEL_ID=XXX  # the channel id in which the bot must write
 LAUNCH_TIME=18:00  # the time at which the timetable should be imported
+SKIP_REST_DAYS=ENABLED # Add this line if you want to skip weekends
```

Comment:
The name `SKIP_REST_DAYS` is a bit ambiguous because the implementation in `bot.py` only skips the weekend days and not weekdays without lessons. I think the name `SKIP_WEEKENDS` is clearer, what is your opinion ?

For the value, I think we can be satisfied with a `TRUE` / `FALSE` management, simpler than `ENABLED` / `DISABLED`.

Reply:
You're right, `SKIP_WEEKENDS` is clearer. What do you think about skipping weekdays without lessons, would be confusing in my opinion but what's yours?

Reply:
Agree with you